sdk_test/v2_client.py

```diff
@@ -0,0 +1,28 @@
+from rapyuta_io.utils import prepend_bearer_to_auth_token, RestClient
+from rapyuta_io.utils.rest_client import HttpMethod
+from rapyuta_io.utils.utils import get_api_response_data
+
+
+class V2Client:
```

Comment:
todo: Add the docstring to explain why this was added here to break cyclic dependency between SDK and CLI.